common/script_values/99_steward_values.txt

```diff
@@ -279,6 +279,10 @@ promote_culture_east_settling_bonus = {
 		multiply = 0.15
 	}
 }
+promote_culture_epidemic_bonus = {
+	value = steward_promote_culture_base_total
+	multiply = 0.5
```

Comment:
+15%
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/103773393-6d7f7580-5044-11eb-8fc4-be3c769c3f07.png)

</details>